index.html

```diff
@@ -3,7 +3,1043 @@
+                    alt="The next web"
+                  />
+                  <h4 class="clicky-header">CLICKY</h4>
+                  <div class="clicky-rate">
+                    <h4 class="clicky-price">$700k</h4>
+                    <h5 class="clicky-fund">Seed fund</h5>
+                  </div>
+                </a>
+                <div class="third-clicky-container">
+                  <a href="#" aria-label="the-next-web" class="clicky-anchor first"
+                    ><i class="icon investment-info-icon icon--tag--dark"></i
+                    ><span>E-Commerce, +2</span>
+                  </a>
+                  <a class="clicky-anchor" href="#">
+                    <i class="icon investment-info-icon icon--briefcase--dark"></i>
+                    <span>Asif Keshodia</span>
+                  </a>
+                </div>
+                <ul class="clicky-ul">
+                  <li>
+                    <a class="clicky-li" href="#">
+                      Pakistani fashion e-commerce startup attracts USD 700,000 in Pre-Series A
+                      funding round<span class="clicky-span">on&nbsp;TechJuice</span></a
+                    >
+                  </li>
+                </ul>
+              </div>
+            </div>
+
+            <div class="clicky-container">
+              <div class="second-clicky-container">
+                <a href="#" aria-label="the-next-web" class="">
+                  <img
+                    class="clicky-image"
+                    src="./image/tnw-image-extra-002.jpeg"
+                    alt="The next web"
+                  />
+                  <h4 class="clicky-header">CLICKY</h4>
+                  <div class="clicky-rate">
+                    <h4 class="clicky-price">$700k</h4>
+                    <h5 class="clicky-fund">Seed fund</h5>
+                  </div>
+                </a>
+                <div class="third-clicky-container">
+                  <a href="#" aria-label="the-next-web" class="clicky-anchor first"
+                    ><i class="icon investment-info-icon icon--tag--dark"></i
+                    ><span>E-Commerce, +2</span>
+                  </a>
+                  <a class="clicky-anchor" href="#">
+                    <i class="icon investment-info-icon icon--briefcase--dark"></i>
+                    <span>Asif Keshodia</span>
+                  </a>
+                </div>
+                <ul class="clicky-ul">
+                  <li>
+                    <a class="clicky-li" href="#">
+                      Pakistani fashion e-commerce startup attracts USD 700,000 in Pre-Series A
+                      funding round<span class="clicky-span">on&nbsp;TechJuice</span></a
+                    >
+                  </li>
+                </ul>
+              </div>
+            </div>
+
+            <div class="clicky-container">
+              <div class="second-clicky-container">
+                <a href="#" aria-label="the-next-web" class="">
```

Comment:
💡 I would search for `class=""` in VSCode and replace it with nothing.

Reply:
Thanks a lot, I really appreciate your feedback!